VWOSdk/Internal/Helper/UuidV5Helper.cs

```diff
@@ -38,16 +38,16 @@ internal class UuidV5Helper
         /// </summary>
         private static readonly Guid IsoOidNamespace = new Guid("6ba7b812-9dad-11d1-80b4-00c04fd430c8");
         private static readonly string file = typeof(UuidV5Helper).FullName;
-        private const string GUID_FORMAT = "N";
-        private const string VWO_NAMESPACE_URL = "https://vwo.com";
+        private const string GuidFormat = "N";

```

Comment:
these are constants, no need to change them to Pascalcase @sahilbathla @AmishaSingla